ts/torch_handler/base_handler.py

```diff
@@ -235,6 +307,8 @@ def handle(self, data, context):
             if self._is_describe():
                 output = [self.describe_handle()]
             else:
+                # What's a better way to do this
+                data = torch.randn(1, 1)
```

Comment:
Looking for feedback on this, not sure how I can curl data that I don't want to be preprocessed

Reply:
Not sure what we are trying to do here?

Reply:
So the base handler does preprocess data because it can't make assumptions about the data type

So I'd like to essentially curl a torch tensor for the inference directly and I'm not sure what the right way to do that is. One workaround is I can inherit from this handler and in my test have a preprocesser but if I put the test here I believe it will be more general and safer